factory/amf_config_test.go

```diff
@@ -0,0 +1,38 @@
+// SPDX-FileCopyrightText: 2024 Open Networking Foundation <info@opennetworking.org>
+// Copyright 2019 free5GC.org
```

Comment:
Is not this a file that you are creating? If so, should it only have a "Canonical" copyright?

Reply:
You are right. I only added "Canonical" copyright now.